src/components/POLine/POLineView.js

```diff
@@ -110,10 +111,14 @@ class POLineView extends Component {
 
   onEditPOLine = (e) => {
     if (e) e.preventDefault();
-    const { queryMutator } = this.props;
+    const { queryMutator, order, line } = this.props;
 
-    queryMutator.update({ layer: 'edit-po-line' });
-  }
+    queryMutator.update({
```

Comment:
please try to avoid query mutator

Reply:
I tried, but case with `history.push` don't work here.